### PR TITLE
Added two global variables to configure project test directory and test file

### PR DESCRIPTION
**Pull request by pmacosta**
**g:pytest_test_dir** (defaults to 'tests'): configures the project test directory
**g:pytest_test_file** (defaults to 'tests.py'): configures the test file

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -274,6 +274,27 @@ This is how a very verbose distributed call would look like::
...
+to be in ``/path/to/project/test``). The global variable ``pytest_test_dir`` may
+be used to change this, for example::
+
+    let g:pytest_test_dir = 'test_suite'
+
+configures the test directory to be ``/path/to/project/test_suite``
+
+Test runner
+^^^^^^^^^^^
+By default the test runner is ``tests.py``.The global variable
```
Comment (alfredodeza) [inline README.rst]: I am not clear on what "test runner" is here. Initially I'd assumed it meant a different executable, but after looking at the code changes, it seems that it is meant to point to a single test file.

Can this variable and the docs references be updated to reflect that this is the test file and doesn't have anything to do with the test runner itself?

**Comment by pmacosta** [comment]
I agree that I did not use the proper terminology. I renamed pytest_test_runner to pytest_test_file, updated code and documentation

**Comment by alfredodeza** [comment]
Awesome! Would you mind squashing the fix? I'll merge once that is done. Thanks again!

**Comment by pmacosta** [comment]
I have squashed the fix, It should be ready to go now

**Pull request merged**